src/oscar/apps/basket/middleware.py

```diff
@@ -7,7 +7,7 @@
 from oscar.core.compat import MiddlewareMixin, user_is_authenticated
 from oscar.core.loading import get_class, get_model
 
-Applicator = get_class('offer.utils', 'Applicator')
+Applicator = get_class('offer.applicator', 'Applicator')
```

Comment:
While this is the proper location for new projects (which left me confused why my `Applicator` was not used), this could break older projects that still place the override in `offer.utils`. Is there some kind of backwards compatibility for class "moves", or should this be mentioned in the release notes?